### DISCUSSION TITLE
Connection refused while running on real ios device

### DISCUSSION THREAD

**Discussion by Ben1980** [supabase/supabase#14611] [Questions]
I have a local dev supabase cli instance running. Connection works fine with Flutter on an ios simulator but with a real device (connected via USB) I get an exception refused exception.
The same device is working well with the supabase staging and production instance online. The URL and anonkey are correct as far as I can tell. How can I find out what exactly the problem is? There is no log of the reason why I get a connection refused exception.

Supabase cli version: 1.63.2
Flutter version: 3.10.2
supabase_flutter version: 1.10.0

Interesting enough, it was working in the past, but I don't know what might cause the issue now.

**Comment by dshukertjr** [comment]
Hi there,

This seems more of a general Flutter question than a Supabase question. `localhost` is a virtual server setup on your computer. Since your real iOS device is not part of your computer, it cannot just access it directly. If you google online, you might find some information on how to connect to local host on your computer from real devices.
